src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -440,6 +440,13 @@ public DamageCause get(final EntityDeathEvent e) {
 			}
 		}, 0);
 		// ProjectileHitEvent
+		EventValues.registerEventValue(ProjectileHitEvent.class, Block.class, new Getter<Block, ProjectileHitEvent>() {
+			@Nullable
+			@Override
+			public Block get(ProjectileHitEvent e) {
+				return e.getHitBlock();
+			}
+		}, 0);
```

Comment:
Needs a version check, this didn't method didn't exist till 1.11

Reply:
Sorry, how would I add that?

Reply:
I changed the comment to 1.11+, is there anything else I need to do?

Reply:
You will need to dynamically check the existence of the method, like done here https://github.com/SkriptLang/Skript/blob/4f0fd027bfd6340f3966e1daf57c6189039f08b3/src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java#L365